on the oadp-1.6 branch
```
whayutin@fedora:~/OPENSHIFT/git/OADP/oadp-operator$ go test ./tests/release/ -v -count=1
=== RUN   TestImageReferencesTagVersion
--- PASS: TestImageReferencesTagVersion (0.00s)
=== RUN   TestImageReferencesMatchCSV
    image_references_test.go:94: image-references tag "oadp-kubevirt-velero-plugin-rhel9" has image "quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6" which is not in CSV RELATED_IMAGE_* env vars
    image_references_test.go:94: image-references tag "oadp-kubevirt-datamover-plugin-rhel9" has image "quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6" which is not in CSV RELATED_IMAGE_* env vars
--- FAIL: TestImageReferencesMatchCSV (0.00s)
=== RUN   TestValidateImageReferencesMatchCSV
```

--- a/bundle/manifests/oadp-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/oadp-operator.clusterserviceversion.yaml
@@ -1481,7 +1481,7 @@ spec:
                 - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_CONTROLLER
                   value: quay.io/konveyor/kubevirt-datamover-controller:latest
                 - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_PLUGIN
-                  value: quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6
+                  value: quay.io/konveyor/kubevirt-datamover-plugin:latest
                 image: quay.io/konveyor/oadp-operator:latest
                 imagePullPolicy: Always
                 livenessProbe:
@@ -1724,6 +1724,6 @@ spec:
     name: vmdp-cli-download
   - image: quay.io/konveyor/kubevirt-datamover-controller:latest
     name: kubevirt-datamover-controller
-  - image: quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6
+  - image: quay.io/konveyor/kubevirt-datamover-plugin:latest
     name: kubevirt-datamover-plugin
   version: 99.0.0

--- a/bundle/manifests/oadp-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/oadp-operator.clusterserviceversion.yaml
@@ -1480,6 +1480,8 @@ spec:
                   value: quay.io/konveyor/oadp-vmdp-binaries:latest
                 - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_CONTROLLER
                   value: quay.io/konveyor/kubevirt-datamover-controller:latest
+                - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
+                  value: quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6
                 image: quay.io/konveyor/oadp-operator:latest
                 imagePullPolicy: Always
                 livenessProbe:
@@ -1701,7 +1703,7 @@ spec:
   - image: quay.io/konveyor/velero-plugin-for-gcp:latest
     name: velero-plugin-for-gcp
   - image: quay.io/konveyor/kubevirt-velero-plugin:latest
-    name: kubevirt-velero-plugin
+    name: openshift-adp-controller-manager-kubevirt-velero-plugin
   - image: quay.io/redhat-user-workloads/ocp-art-tenant/oadp-hypershift-oadp-plugin-main:main
     name: hypershift-velero-plugin
   - image: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
@@ -1722,4 +1724,6 @@ spec:
     name: vmdp-cli-download
   - image: quay.io/konveyor/kubevirt-datamover-controller:latest
     name: kubevirt-datamover-controller
+  - image: quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6
+    name: openshift-adp-controller-manager-kubevirt-velero-plugin
   version: 99.0.0

--- a/bundle/manifests/oadp-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/oadp-operator.clusterserviceversion.yaml
@@ -1480,8 +1480,8 @@ spec:
                   value: quay.io/konveyor/oadp-vmdp-binaries:latest
                 - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_CONTROLLER
                   value: quay.io/konveyor/kubevirt-datamover-controller:latest
-                - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
-                  value: quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6
+                - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_PLUGIN
+                  value: quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6
                 image: quay.io/konveyor/oadp-operator:latest
                 imagePullPolicy: Always
                 livenessProbe:
@@ -1703,7 +1703,7 @@ spec:
   - image: quay.io/konveyor/velero-plugin-for-gcp:latest
     name: velero-plugin-for-gcp
   - image: quay.io/konveyor/kubevirt-velero-plugin:latest
-    name: openshift-adp-controller-manager-kubevirt-velero-plugin
+    name: kubevirt-velero-plugin
   - image: quay.io/redhat-user-workloads/ocp-art-tenant/oadp-hypershift-oadp-plugin-main:main
     name: hypershift-velero-plugin
   - image: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
@@ -1724,6 +1724,6 @@ spec:
     name: vmdp-cli-download
   - image: quay.io/konveyor/kubevirt-datamover-controller:latest
     name: kubevirt-datamover-controller
-  - image: quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6
-    name: openshift-adp-controller-manager-kubevirt-velero-plugin
+  - image: quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6
+    name: kubevirt-datamover-plugin
   version: 99.0.0

--- a/config/manager/manager.yaml
+++ b/config/manager/manager.yaml
@@ -103,7 +103,7 @@ spec:
             - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_CONTROLLER
               value: quay.io/konveyor/kubevirt-datamover-controller:latest
             - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_PLUGIN
-              value: quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6
+              value: quay.io/konveyor/kubevirt-datamover-plugin:latest
           args:
             - --leader-elect
           image: controller:latest

--- a/config/manager/manager.yaml
+++ b/config/manager/manager.yaml
@@ -102,6 +102,8 @@ spec:
               value: quay.io/konveyor/oadp-vmdp-binaries:latest
             - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_CONTROLLER
               value: quay.io/konveyor/kubevirt-datamover-controller:latest
+            - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
+              value: quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6
           args:
             - --leader-elect
           image: controller:latest

--- a/config/manager/manager.yaml
+++ b/config/manager/manager.yaml
@@ -102,8 +102,8 @@ spec:
               value: quay.io/konveyor/oadp-vmdp-binaries:latest
             - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_CONTROLLER
               value: quay.io/konveyor/kubevirt-datamover-controller:latest
-            - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
-              value: quay.io/konveyor/kubevirt-velero-plugin:oadp-1.6
+            - name: RELATED_IMAGE_KUBEVIRT_DATAMOVER_PLUGIN
+              value: quay.io/konveyor/kubevirt-datamover-plugin:oadp-1.6
           args:
             - --leader-elect
           image: controller:latest

--- a/go.mod
+++ b/go.mod
@@ -42,6 +42,7 @@ require (
 	github.com/vmware-tanzu/velero v1.14.0
 	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
 	google.golang.org/api v0.256.0
+	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/klog/v2 v2.130.1
 )
 
@@ -193,7 +194,6 @@ require (
 	google.golang.org/protobuf v1.36.10 // indirect
 	gopkg.in/evanphx/json-patch.v4 v4.12.0 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
-	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/cli-runtime v0.33.3 // indirect
 	k8s.io/kube-openapi v0.0.0-20250318190949-c8a335a9a2ff // indirect
 	sigs.k8s.io/json v0.0.0-20241010143419-9aa6b5e7a4b3 // indirect

